delicious-homepage.html

```diff
@@ -27,31 +31,35 @@
         <!-- Primary Page Layout-->
 
     <header>
-        <img src="logo.jpg" alt="Delicious" />
-        <h1><a href="http://delcious.com">Delicious</h1>
-        <p>The best food blog on the web.</p>
-        <ul class "social-links">
-            <li><a href="http://www.facebook.com/"></a></li>
-            <li><a href="http://www.twitter.com/"></a></li>
-            <li><a href="http://www.google.com/plus"></a></li>
-            <li><a href="http://www.instagram.com/"></a></li>
-            <li><a href="http://www.flickr.com/"></a></li>
-            <li><a href="http://www.pinterest.com/"></a></li>
-            <li><a href="http://www.deliciousrss.xml/"></a></li>
-            <li><a href="http://www.delicious.com/subscribe/"></a></li>
-        </ul>
+        <section class="blog">
+                    <img src="logo.jpg" width="45px" class="logo"/>
+            <h1><a href="http://delicious.com">Delicious</a>
+            <strong>The best food blog on the web.</strong></h1>
+
+                <ul class="social clearfix">
+                    <a href="#" title="Facebook"><img src="facebook.jpg" width="24px"></a>
+                    <a href="#" title="Twitter"><img src="facebook.jpg" width="24px"></a>
+                    <a href="#" title="Google+"><img src="facebook.jpg" width="24px"></a>
+                    <a href="#" title="Instagram"><img src="facebook.jpg" width="24px"></a>
+                    <a href="#" title="Flickr"><img src="facebook.jpg" width="24px"></a>
+                    <a href="#" title="Pinterest"><img src="facebook.jpg" width="24px"></a>
+                    <a href="#"> <img src="facebook.jpg" width="24px"></a>
+                    <a href="#"> <img src="facebook.jpg" width="24px"></a>
+                </ul>
+
     </header>
-    <section>
-        <h2>Chocolate Pizza</h2>
-        <p>Posted on 15 Dec 2013 / Desserts</p>
-        <button>Print</button>
-        <article>
-            <img src="http://picturestuff.com"/>
+        <div class="divider"></div>
+    <section class="main">
+        <h2>Chocolate Pizza
+        <strong>Posted on 15 Dec 2013 /<span class="type"> Desserts</span>
```

Comment:
Need a closing strong tag. Also, conisder whether the use of the tag is needed at all.